src/librustc_target/spec/riscv32gc_unknown_linux_gnu.rs

```diff
@@ -0,0 +1,25 @@
+use crate::spec::{CodeModel, LinkerFlavor, Target, TargetOptions, TargetResult};
+
+pub fn target() -> TargetResult {
+    Ok(Target {
+        llvm_target: "riscv32-unknown-linux-gnu".to_string(),
+        target_endian: "little".to_string(),
+        target_pointer_width: "32".to_string(),
+        target_c_int_width: "32".to_string(),
+        target_env: "gnu".to_string(),
+        data_layout: "e-m:e-p:32:32-i64:64-n32-S128".to_string(),
+        arch: "riscv32".to_string(),
+        target_os: "linux".to_string(),
+        target_vendor: "unknown".to_string(),
+        linker_flavor: LinkerFlavor::Gcc,
+        options: TargetOptions {
+            abi_blacklist: super::riscv_base::abi_blacklist(),
```

Comment:
```
error[E0425]: cannot find function `abi_blacklist` in module `super::riscv_base`
  --> src/librustc_target/spec/riscv32gc_unknown_linux_gnu.rs:16:47
   |
16 |             abi_blacklist: super::riscv_base::abi_blacklist(),
   |                                               ^^^^^^^^^^^^^ not found in `super::riscv_base`

    Checking rustc_query_system v0.0.0 (/checkout/src/librustc_query_system)
    Checking chalk-engine v0.14.0
error[E0560]: struct `spec::TargetOptions` has no field named `abi_blacklist`
  --> src/librustc_target/spec/riscv32gc_unknown_linux_gnu.rs:16:13
   |
16 |             abi_blacklist: super::riscv_base::abi_blacklist(),
   |             ^^^^^^^^^^^^^ `spec::TargetOptions` does not have this field
   |
   = note: available fields are: `is_builtin`, `linker`, `lld_flavor`, `pre_link_args`, `pre_link_objects` ... and 80 others
```
https://github.com/rust-lang/rust/blob/master/RELEASES.md#compatibility-notes:
> The target configuration option abi_blacklist has been renamed to unsupported_abis. The old name will still continue to work.
```suggestion
            unsupported_abis: super::riscv_base::unsupported_abis(),
```